assignment-client/src/assets/AssetServer.cpp

```diff
@@ -49,8 +50,20 @@ void AssetServer::run() {
     auto nodeList = DependencyManager::get<NodeList>();
     nodeList->addNodeTypeToInterestSet(NodeType::Agent);
 
-    _resourcesDirectory = QDir(QCoreApplication::applicationDirPath()).filePath("resources/assets");
+    const QString OLD_RESOURCES_PATH = "resources/assets";
+    const QString RESOURCES_PATH = "assets";
+
+    _resourcesDirectory = QDir(ServerPathUtils::getDataFilePath(RESOURCES_PATH));
     if (!_resourcesDirectory.exists()) {
+        qDebug() << "Asset resources directory not found, searching for existing asset resources";
+        QString oldDataDirectory = QCoreApplication::applicationDirPath();
+        auto oldResourcesDirectory = QDir(oldDataDirectory).filePath(OLD_RESOURCES_PATH);
+
+        if (QDir(oldResourcesDirectory).exists()) {
+            qDebug() << "Existing assets found in " << oldResourcesDirectory << ", copying to " << _resourcesDirectory;
+            QFile::copy(oldResourcesDirectory, _resourcesDirectory.absolutePath());
```

Comment:
what will this do if the directory doesn't exist? I think we need a call to `QDir().mkPath()` for the containing dir first